src/actions/actionTypes.js

```diff
@@ -4,3 +4,8 @@ export const RECEIVE_STUFF = 'RECEIVE_STUFF'
 // edit
 export const EDIT_MODE_ENABLED = 'EDIT_MODE_ENABLED'
 export const EDIT_MODE_DISABLED = 'EDIT_MODE_DISABLED'
+
+// map_show
+export const MAPS_LOADED = 'MAPS_LOADED'
+export const MAP_CREATED = 'MAP_CREATED'
+export const MAP_REMOVED = 'MAP_REMOVED'
```

Comment:
please follow the naming convention: https://hack4impact.slack.com/archives/C9B26B83W/p1522348147000286